tuning/trainercontroller/operations/hfcontrols.py

```diff
@@ -10,6 +10,9 @@
 # Local
 from .operation import Operation
 
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
```

Comment:
Why is the default log level for the trainer controller debug? In general it's warning to print minimal print messages and the user can increase the level as needed